### DISCUSSION TITLE
优先答复已Star本项目用户的提问

### DISCUSSION THREAD

**Discussion by zouyaoji** [zouyaoji/vue-cesium#245] [Show and tell]
Due to busy affairs and limited time and energy, all questions and issues cannot be dealt with in a timely manner.
Therefore, it is hereby declared that the questions that have been starred in this project will be answered first (regardless of whether the question is sooner or later);
Users who use this project and ask questions, please star this project as support, thank you!

由于事务繁忙，时间精力有限，所有疑问及issue不能全部及时处理，
所以特此声明，对于已star本项目的提问，将优先答复(不论提问早晚)；
使用本项目并提问的用户，请star下本项目作为支持，谢谢！

**Comment by zouyaoji** [comment]
感谢理解

**Reply by cweibo** [reply]
理解理解，大佬辛苦

**Reply by zouyaoji** [reply]
你是托吗

**Reply by NanF007** [reply]
已star 求作者回复715
